src/docGenerator.ts

```diff
@@ -102,6 +106,14 @@ export class DocGenerator {
         descriptionParameters
       )
     })
+
+    var extraPatternsDescriptions = flatMap(extraPatternEntries, (entry) => {
```

Comment:
This can be a `const`:
```suggestion
    const extraPatternsDescriptions = flatMap(extraPatternEntries, (entry) => {
```